scheduler/src/cook/mesos/sandbox.clj

```diff
@@ -28,77 +28,70 @@
             [metrics.meters :as meters]
             [metrics.timers :as timers]))
 
-(def sandbox-aggregator-message-rate (meters/meter ["cook-mesos" "scheduler" "sandbox-aggregator-message-rate"]))
-(def sandbox-aggregator-pending-count (counters/counter ["cook-mesos" "scheduler" "sandbox-aggregator-pending-count"]))
-(def sandbox-pending-sync-host-count (counters/counter ["cook-mesos" "scheduler" "sandbox-pending-sync-host-count"]))
-(def sandbox-updater-pending-entries (histograms/histogram ["cook-mesos" "scheduler" "sandbox-updater-pending-entries"]))
-(def sandbox-updater-publish-duration (timers/timer ["cook-mesos" "scheduler" "sandbox-updater-publish-duration"]))
-(def sandbox-updater-publish-rate (meters/meter ["cook-mesos" "scheduler" "sandbox-updater-publish-rate"]))
-(def sandbox-updater-tx-duration (timers/timer ["cook-mesos" "scheduler" "sandbox-updater-tx-duration"]))
-(def sandbox-updater-tx-rate (meters/meter ["cook-mesos" "scheduler" "sandbox-updater-tx-rate"]))
-(def sandbox-updater-unprocessed-count (counters/counter ["cook-mesos" "scheduler" "sandbox-updater-unprocessed-count"]))
-(def sandbox-updater-unprocessed-entries (histograms/histogram ["cook-mesos" "scheduler" "sandbox-updater-unprocessed-entries"]))
-
-(defn agent->task-id->sandbox
-  "Returns the sandbox in the current agent state."
-  [task-id->sandbox-agent task-id]
-  (get @task-id->sandbox-agent task-id))
-
 (defn clear-agent-state
   "Clears the published entries from the aggregated state of the agent.
-   Since we expect instances to have the same sandbox, we do not check for value equality."
-  [task-id->sandbox published-task-id->sandbox]
-  (let [task-id->sandbox' (apply dissoc task-id->sandbox (keys published-task-id->sandbox))]
-    (counters/clear! sandbox-aggregator-pending-count)
-    (counters/inc! sandbox-aggregator-pending-count (count task-id->sandbox'))
-    task-id->sandbox'))
+   Since we expect instances to have the same field value, we do not check for value equality."
+  [task-id->field-value field-name published-task-id->field-value]
```

Comment:
What about just `value` instead of `field-value` for both arguments?

Reply:
Renamed `field-value` to `value`.